cfme/tests/ssui/test_ssui_myservice.py

```diff
@@ -52,15 +65,70 @@ def test_myservice_crud(appliance, setup_provider, context, order_catalog_item_i
         my_service.delete()
 
 
-@pytest.mark.uncollectif(lambda: current_version() < '5.8')
+@pytest.mark.uncollectif(lambda: current_version() < '5.8' or current_version() > '5.9')
```

Comment:
**Question**: What versions should this test run for? 5.8, 5.9 and ignore upstream?

Reply:
Hey @rlbabyuk  it should run for 5.8 only, support for 5.7 will be added in next sprints and then 5.9.

Reply:
@kedark3 please update current_version() >= '5.9', if 5.9 should be skipped

Reply:
But then it will strictly just do == '5.9' right? shouldn't it be >= 5.9?